UnityProject/Assets/Scripts/Doors/ShutterController.cs

```diff
@@ -84,15 +84,15 @@ public void SetLayer(int layer, int sortingLayer)
 		}
 	}
 
-	[Server]
-	private void DamageOnClose()
-	{
-		IEnumerable<HealthBehaviour> healthBehaviours = _matrix.Get<HealthBehaviour>(_registerTile.Position);
-		foreach (HealthBehaviour healthBehaviour in healthBehaviours)
-		{
-			healthBehaviour.ApplyDamage(gameObject.name, 500, DamageType.BRUTE);
-		}
-	}
+    [Server]
+    private void DamageOnClose()
+    {
+        IEnumerable<HealthBehaviour> healthBehaviours = _matrix.Get<HealthBehaviour>(_registerTile.Position);
+        foreach (HealthBehaviour healthBehaviour in healthBehaviours)

```

Comment:
I see spaces here. DO NOT WANT SPACES.
TABS for LIFE!

Reply:
rider default settings bro :D

Reply:
We don't use those, we have custom settings in regards to codestyle and a custom TODO filter for the FIXME tags

Those are included in /tools/

Reply:
noted for next time, Will the auto cleaner catch this when its run again?

Reply:
Yeah that will be fine :)